index.mjs

```diff
@@ -1,10 +1,16 @@
-import t from 'typical/index.mjs'
-import flatten from 'reduce-flatten'
-
 /**
  * @module wordwrapjs
  */
 
+/**
```

Comment:
Could you do me a favour please and review your jsdoc changes. Try to fix them so 

1. The `npm run docs` output looks accurate.. If you run it now, "inner" variables and identifiers are exposed in the README documentation - only the public API should be documented in the README, not inner functionality.. 
2. It still works well with Typescript

Reply:
If you can't get the API docs looking correct, or if you are just short of time, please revert the jsdoc comments to what they were before.

Reply:
> If you run it now, "inner" variables and identifiers are exposed in the README documentation - only the public API should be documented in the README, not inner functionality..

Would have been nice if the doc generator where context aware.

```js
// public
export class Foo {

  /** 
   * @param {number} a
   * @param {number} b
   */
  add(a,b) { this.#debug('logging') }

  /** 
   * @param {any} x
   */
  #log (x) {
    console.log(x)
  }

}

/** @type {...} ... */
function notExported() { }
```